crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -549,7 +563,17 @@ async fn fetch_blocks_at_celestia_height(
         .in_current_span()
         .collect()
         .await;
-    Ok(blocks)
+    let reconstructed = ReconstructedBlocks {
+        celestia_height,
+        sequencer_namespace,
+        rollup_namespace,
+        blocks,
+    };
+    info!(
```

Comment:
Emitting this at info level because this strikes me as the main functionality of conductor (receiving new blocks).

Or should the level be stepped down?